src/lib/components/coverage-file-table-row.jsx

```diff
@@ -38,7 +38,7 @@ export default function FlowCoverageFileTableRow(
     threshold
   } = props;
 
-  const isFlow = annotation === 'flow';
```

Comment:
@zeorin locally I'm turning this check into a shared helper (because we are doing the same check in multiple places and we want to be sure that we don't miss one if/when we are going to change this again).

Also what do you think if I reverse the check?

My guess is that the flow team is never going to add an additional annotation that is less strict of "flow", and so we could just check that the annotation retrieved from a file is not one of "no flow" or "flow weak" (and hopefully we will never have to change this check again, "crossed fingers" here ;-))

Reply:
I should have added the comment on the next line :-P 
(anyway, this was basically one of the three `['flow', 'flow strict', 'flow strict-local'].indexOf(annotation)` checks).

Reply:
Yes, a `isFlowAnnotation(annotation)` function would be better, that way there's a single source of truth for the check logic.
However, I would not assume that if the pragma isn't `no flow` or `flow weak` that it's the pragma we'd consider valid. What if the user makes a typo (e.g. `folw` or `floww`), or what if the pragma we find has nothing to do with flow (`prettier` and `format` are already in use), or what if they introduce another flow pragma that has different implications for coverage purposes? Better to be safe and check explicitly for the expected known-good values.

Reply:
@zeorin sounds good to me too, in any case I would not expect flow to define too many new annotations, and so moving that check into its own helper function should be more then enough to make it less error prone to add support for more of these annotations (when/if they do add them).